edxval/tests/test_views.py

```diff
@@ -781,6 +778,12 @@ def test_update_auto_generated_images(self):
             'post_data': {'course_id': 'test_course_id', 'edx_video_id': 'super-soaker', 'generated_images': [1, 2, 3]},
             'message': "list must only contain strings.']"
         },
+        {
+            'post_data': {
+                'course_id': 'test_course_id', 'edx_video_id': 'super-soaker', 'generated_images': [1, 2, 3, 4]
```

Comment:
```
This validation is already exists in model so checking it
Another reason for this check is in django2 with py3 validation are breaking. So it will help the further debugging in py3+dj20 fixes.
```